.storybook/recipes/Filters/Filters.tsx

```diff
@@ -1,31 +1,43 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import FiltersDrawer from '../../../src/components/FiltersDrawer';
 import {
   FiltersPopover,
   FiltersPopoverProps,
 } from '../../../src/components/FiltersPopover/FiltersPopover';
 
+import breakpoint from '../../../src/design-tokens/tier-1-definitions/breakpoints';
 type Props = {
-  /**
-   * Filters component to render.
-   */
-  as?: 'FiltersDrawer' | 'FiltersPopover';
   /**
    * Popover placement options relative to the filters trigger button.
    */
   placement?: FiltersPopoverProps['placement'];
 } & FiltersPopoverProps;
 
 /**
- * Primary UI component for user interaction
+ * Demonstrates usage of both FiltersPopover and FiltersDrawer depending on screensize.
  */
-export const Filters = ({ as, placement, ...other }: Props) => {
+export const Filters = ({ placement, ...other }: Props) => {
+  const [isLarge, setIsLarge] = useState(false);
+  const popoverBreakpoint = parseInt(breakpoint['eds-bp-md']) * 16; /* 1 */
+  useEffect(() => {
+    updateScreenSize();
+    window.addEventListener('resize', updateScreenSize);
+    return () => {
+      window.removeEventListener('resize', updateScreenSize);
+    };
+  });
+
+  const updateScreenSize = () => {
+    if (window.innerWidth >= popoverBreakpoint) {
+      setIsLarge(true);
+    } else {
+      setIsLarge(false);
+    }
+  };
```

Comment:
I'm wondering if we should add debouncing and only update state if it needs to be updated to demonstrate best practices in case someone decides to copy/paste this into the downstream product.

Reply:
Good call, will add debouncing

Reply:
Added debouncing and conditional, still working:


https://user-images.githubusercontent.com/86632227/191861400-28119327-ae79-4e17-9916-5cd0b723316b.mov